packages/webapp/src/pages/Environment/Settings.tsx

```diff
@@ -118,6 +124,9 @@ export const EnvironmentSettings: React.FC = () => {
         setSlackConnectedChannel(slack_notifications_channel);
 
         setEnvVariables(env_variables);
+
+        setOtlpEndpoint(environment.otlp_settings?.endpoint || '');
+        setOtlpHeaders(environment.otlp_settings?.headers || {});
```

Comment:
caveat: headers are reordered by key automatically. I didn't want to make the model and code more complicated for such a tiny feature.